### DISCUSSION TITLE
is there way to disabling bundling of SCTP packets

### DISCUSSION THREAD

**Discussion by rajukasaramoni** [sctplab/usrsctp#686] [Q&A]
Please help with way to disable bundling of SCTP packets.

**Comment by tuexen** [comment]
No. But why do you want to do this? If you want to avoid any additional delay in sending user messages, you can disable the Nagle algorithm. See [RFC 6458](https://datatracker.ietf.org/doc/html/rfc6458#section-8.1.5).
